src/i18n/sv/common.json

```diff
@@ -337,12 +337,9 @@
       "alt": "Helsingfors logotyp"
     },
     "text": "Tomtansökan",
-    "customerService": "Stadsmiljösektorns kundservice",
-    "some": {
-      "facebook": "Helsingfors stad på Facebook",
-      "twitter": "Helsingfors stad på X",
-      "instagram": "Helsingfors stad på Instagram",
-      "linkedin": "Helsingfors stad på LinkedIn"
+    "customerService": {
+      "label": "Stadsmiljösektorns kundservice",
+      "url": "https://www.hel.fi/sv/stadsmiljo-och-trafik/stadsmiljosektorns-kundtjanst"
     },
     "moveToTop": "Gå upp",
     "accessabilityStatement": {
```

Comment:
This isn't broken or part of the changes, but this variable name has a typo. Should be `accessibilityStatement`.

Reply:
@juho-kettunen-nc fixed, could yo ure-review?